Changelog.md

```diff
@@ -4,8 +4,14 @@ Changelog
 
 Current master
 --------------
+- Remove RxAtomic references
+
+4.0.0
+-------
 - Add `completions` property to `CompletableAction`
 - Change `inputs` type to `AnyObserver<Input>`
+- Added full support for Swift 5.0
+- Added full support for RxSwift 5.0
```

Comment:
Did we do this in the PR as well? Or was it missing?

Reply:
Oh, yeah it was already done. I noticed that it was missing from the changelog so I added it in. I moved it under `Current master`.